.github/settings.yml

```diff
@@ -14,11 +14,7 @@ repository:
 collaborators: []
 
 teams:
-  - name: central
-    permission: push
-  - name: happy
-    permission: push
-  - name: hydra
```

Comment:
@timotree3 - are you sure we should be removing hydra here?

Reply:
I added it back